rep-2000.rst

```diff
@@ -766,41 +765,41 @@ Minimum language requirements:
-| PCL         |     1.10.0     |              N/A               |       1.11.1     |       1.10.0        |
-+-------------+----------------+----------------+---------------+------------------+---------------------+
-|                                **RMW DDS Middleware Providers**                                        |
-+-------------+----------------+----------------+---------------+------------------+---------------------+
-| Cyclone DDS |                                     0.7.x (Coquette)                                     |
-+-------------+----------------+----------------+---------------+------------------+---------------------+
-| Fast-DDS    |                                       2.3.x                                              |
-+-------------+----------------+----------------+---------------+------------------+---------------------+
-| Connext DDS |                              5.3.1              |                 N/A                    |
-+-------------+----------------+----------------+---------------+------------------+---------------------+
++-------------+---------------------------------+-------------------------------------------------------------------+
+|             |        Required Support         |                        Recommended Support                        |
++-------------+----------------+----------------+---------------+----------+------------------+---------------------+
+| Package     |  Ubuntu Focal  |  Windows 10**  |    MacOS**    |  RHEL 8  | Debian Bullseye  |   OpenEmbedded**    |
++=============+================+================+===============+==========+==================+=====================+
+| CMake       |     3.16.3     |     3.19.1     |    3.14.4     |  3.11.4  |      3.18.4      | 3.16.1 / 3.12.2**** |
++-------------+----------------+----------------+---------------+----------+------------------+---------------------+
+| EmPY        |                                                3.3.2                                                |
++-------------+----------------+----------------+---------------+----------+------------------+---------------------+
+| Gazebo      |    11.0.0*     |       N/A      |    11.0.0     |   N/A    |      11.0.0*     |         N/A         |
++-------------+----------------+----------------+---------------+----------+------------------+---------------------+
+| Ignition    |    Edifice*    |       N/A      |   Edifice*    |   N/A    |     Edifice*     |         N/A         |
++-------------+----------------+----------------+---------------+----------+------------------+---------------------+
+| Ogre        |                                     1.10*                                     |         N/A         |
++-------------+----------------+----------------+---------------+----------+------------------+---------------------+
+| OpenCV      |     4.2.0      |     3.4.6*     |     4.2.0     |  3.4.6   |      4.5.1       |  4.1.0 / 3.2.0****  |
```

Comment:
This is minor, but should we make this a triple-star (`***`) instead?  Same goes below, and for the footer at the end of the table.

Reply:
I'm indifferent. You could argue that it might be confusing if the meaning changes between tables.

Either way, I'd consider it a refactor change. Feel free to open a PR to discuss it :)